minibot/new_minibot.py

```diff
@@ -9,6 +9,8 @@
 import ast
```

Comment:
do we ever use this module?

Reply:
oh actually I think we might be using it for creating the bot_script.py program file.